src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/actions.tsx

```diff
@@ -79,9 +79,7 @@ export async function getAllProfileScans(onerepProfileId: number) {
   if (
     !session?.user?.email ||
     !isAdmin(session.user.email) ||
-    // only allow admins to trigger a scan for their own profile in production
-    (process.env.APP_ENV === "production" &&
-      session.user.subscriber?.onerep_profile_id !== onerepProfileId)
```

Comment:
Hmm, this seems unrelated to the description of the PR? Do we want admins to be able to trigger scans for _anyone_ on _prod_?

Reply:
I’ve made this condition stricter so that `getAllProfileScans` always returns a `404` in `prod` on the line below.